### ISSUE TITLE
Net_SFTP has a deprecated constructor

### ISSUE THREAD

**Issue by HtmHell** [LaravelCollective/remote#26]
> Methods with the same name as their class will not be constructors in a future version of PHP; Net_SFTP has a deprecated constructor

I get this error message because this package is using an old version of phpseclib.
http://stackoverflow.com/questions/36973861/laravel-5-1-net-sftp-has-a-deprecated-constructor

Can you fix this?
Thanks in advanced.

**Comment by ikari7789** [comment]
I merged this fix in a while ago, could you please try updating the latest version of the package or check if you're requiring an old version of phpseclib via another package?

**Comment by mlantz** [comment]
Thanks @ikari7789 for jumping in on that :)

**Comment by HtmHell** [comment]
@ikari7789  I have the latest version of 5.1, on my composer.json:
`"laravelcollective/remote": "5.1.*"`
And still having this issue after composer update...

**Comment by ikari7789** [comment]
Well the latest release for 5.1.x requires phpseclib >= 2.0 which has the updated namespaces. The PHPdoc in SecLibGateway.php could be updated, but that doesn't directly affect the code. Could you maybe post a copy of your Composer dependencies and the code that is triggering this warning?

**Comment by HtmHell** [comment]
```
\Config::set('remote.connections.dl.host', $server->ip);
\SSH::into('dl')->run($commands);
```
>         "php": ">=5.5.9",
        "laravel/framework": "5.1.*",
        "guzzlehttp/guzzle": "^6.1",
        "webpatser/laravel-countries": "1.4.3",
        "graham-campbell/throttle": "^5.0",
        "vinkla/hashids": "^2.2",
        "torann/geoip": "0.2.2",
        "doctrine/dbal": "^2.5",
        "greggilbert/recaptcha": "dev-master",
        "illuminate/html": "5.*",
        "laravel/socialite": "^2.0",
        "schuppo/password-strength": "~1.0",
        "jenssegers/agent": "^2.3",
        "predis/predis": "^1.0",
        "laravelcollective/remote": "5.1.*",
        "tymon/jwt-auth": "0.5.*",
        "dingo/api": "1.0.0-beta2",
        "yajra/laravel-datatables-oracle": "~5.0"

Thank you very much for your help!

**Comment by ikari7789** [comment]
I just tried checking out that dependency list and for some reason it is grabbing 5.1.2 for ```laravelcollective/remote```, even though the latest version is 5.1.6. When I tried changing the required version to ```5.1.6``` instead of ```5.1.*```, composer didn't like it to me. It seems that ```namshi/jose``` requires ```phpseclib/phpseclib ~0.3```, which is required by ```tymon/jwt-auth```. In order to fix this you're going to have to bother the author of ```tymon/jwt-auth``` to update the version of ```namshi/jose``` they're requiring.
